ssl/src/main/java/org/wildfly/security/ssl/SNISSLContext.java

```diff
@@ -17,6 +17,9 @@
 
 import javax.net.ssl.SSLContext;
 
+/**
+ * A class which sets the SSLContext based on the given SNIContextMatcher using its default context and protocol.
```

Comment:
@fazer1929 This is not completely right. This class does not set but rather represents SSLContext. Something similar to the following could be used instead: `SSLContext that uses provided SNIContextMatcher to resolve which SSLContext to use for the connection`